packages/create-next-app/index.ts

```diff
@@ -88,17 +88,17 @@ async function run(): Promise<void> {
   }
 
   if (!projectPath) {
-    console.log()
-    console.log('Please specify the project directory:')
-    console.log(
-      `  ${chalk.cyan(program.name())} ${chalk.green('<project-directory>')}`
-    )
-    console.log()
-    console.log('For example:')
-    console.log(`  ${chalk.cyan(program.name())} ${chalk.green('my-next-app')}`)
-    console.log()
     console.log(
-      `Run ${chalk.cyan(`${program.name()} --help`)} to see all options.`
+      `
+      
+      Please specify the project directory:
+        ${chalk.cyan(program.name())} ${chalk.green('<project-directory>')}
```

Comment:
These lines will print with a pretty long offset. This might help in the future: https://github.com/tc39/proposal-string-dedent

Might be worth leaving as is for now.